cmd/s2i/s2i.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/openshift/source-to-image/pkg/cmd/cli"
+)
+
+func main() {
+	rand.Seed(time.Now().UTC().UnixNano())
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
```

Comment:
I don't believe this is necessary - is there a specific reason for adding it out of interest?